roles/monitoring_server/templates/prometheus_rules.yml.j2

```diff
@@ -13,44 +13,44 @@ groups:
     rules:
       - alert: EndpointDown
         expr: probe_success == 0
-        for: 1m
+        for: 2m
```

Comment:
```suggestion
        for: 5m
```

Reply:
This is the one that is causing a lot of spurious alerts. Not sure but it may be due to using `probe_success` rather than alerting on a specific HTTP status code.